src/province/mod.rs

```diff
@@ -4,7 +4,7 @@ use city::*;
 use phf::phf_map;
 use std::fmt;
 
-pub static PROVINCES: phf::Map<&'static str, Province> = phf_map! {
```

Comment:
My goal was to make this module available to the user library So I think it's better to stay public